cwms-data-api/src/test/java/cwms/cda/data/dao/binarytimeseries/TimeSeriesBinaryDaoTest.java

```diff
@@ -0,0 +1,113 @@
+package cwms.cda.data.dao.binarytimeseries;
+
+
+import static cwms.cda.data.dao.DaoTest.getConnection;
```

Comment:
Aren't we trying to move away from this. Needs the CDASEtup... extendswith anyways to make sure it's actually started and you can pull the cwms database testcontainer instance out of that.

Reply:
Oops.  I didn't mean to submit that.